src/main.js

```diff
@@ -121,18 +127,21 @@ module.exports = {
           return Promise.resolve([])
         }
         const filePath = textEditor.getPath()
+        const projectPath = atom.project.relativizePath(filePath)[0] || ''
+        const config = atom.config.get('linter-eslint')
```

Comment:
As this isn't being used elsewhere I don't really see a reason to define it as an explicit variable. (Passing things properly is a longer term goal of mine...)